src/libsyntax/ext/base.rs

```diff
@@ -34,6 +34,12 @@ pub enum Annotatable {
     ForeignItem(P<ast::ForeignItem>),
     Stmt(P<ast::Stmt>),
     Expr(P<ast::Expr>),
+    Arg(ast::Arg),
```

Comment:
Seems like these new ones should also use a `P<..>`?

Reply:
Not in general, fragments here should mirror what is used in AST.